_docs/layout/page-layouts.md

```diff
@@ -115,26 +107,61 @@ Build the content area using a 12 Column Grid. See [grid](/layout/grid) for more
 #### Main content area
 
 - 9 columns
-- Background color: Gray 25, #FAFAFA
+- Background color: Gray 25, `#FAFAFA`
 
 #### Left side rail
 - 3 columns
-- Background color: Gray 100, #EEEEEE
+- Background color: Gray 100, `#EEEEEE`
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Left side rail with top navigation"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/layout/layout-left-side-rail-top-nav.svg)
+![]({{site.url}}/assets/images/layout/page-layouts/layout-left-side-rail-top-nav.png){:width="800px"}
 {% endfigure %}
 
 {% figure [caption:"Left side rail with side navigation"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/layout/layout-left-side-rail-side-nav.svg)
+![]({{site.url}}/assets/images/layout/page-layouts/layout-left-side-rail-side-nav.png){:width="800px"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 </section>
 
+<section class="static-section"  markdown="1">
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Dual panel content area
+
+- Dual Layout enables two variable width panels, entitled primary and secondary, to operate adjacent to one another. User selections or interactions with the primary panel facilate information displayed on secondary panels. Although data is correlated between each panel, panels can scroll independently. Panel widths are defined through application implementation and cannot adjusted manually by the user.
+  - Panels are separated by a 1px solid line. Be sure to use our default canvass color #fafafa as your base.
+  - Below are suggested column breakdowns for dual panel implementation, but can be adjusted based on the density of information being displayed to ensure each panel is optimized for readablility. For additional information see the [helix-ui source code](https://rackerlabs.github.io/helix-ui/components/layouts/horizontal-layout-template.html)
```

Comment:
Revise: "Use the following suggested column breakdowns for dual panel implementation. These columns can be adjusted based on the density of information displaying so each panel is optimized for readability."

Reply:
updated